src/screen/dashboard/sidebar.rs

```diff
@@ -683,10 +672,24 @@ fn upstream_buffer_button<'a>(
             } else {
                 theme::text::error
             }),
-            text(server.to_string())
-                .style(buffer_title_style)
-                .font_maybe(buffer_title_font)
-                .shaping(text::Shaping::Advanced)
+            if let Some(network) = &server.network {
+                Element::from(row![
+                    text(network.name.to_string())
+                        .style(buffer_title_style)
+                        .font_maybe(buffer_title_font.clone())
+                        .shaping(text::Shaping::Advanced),
+                    Space::with_width(6),
+                    text(server.name.to_string())
+                        .style(theme::text::tertiary)
```

Comment:
lets do `theme::text::secondary` here as i treat `tertiary` more as a "hey look at this" where secondary is is used more as a "info".